server/src/main/java/com/flipkart/grayskull/audit/utils/SanitizingObjectMapper.java

```
@@ -27,4 +34,15 @@ public static ObjectMapper create() {
 
         return mapper;
     }
+
+    /**
+     * puts the {@code value} in {@code map} with {@code key} after sanitizing and serializing {@code value} as json
+     */
+    public static void addToMap(Map<String, String> map, String key, Object value) {
+        try {
+            map.put(key, MASK_OBJECT_MAPPER.writeValueAsString(value));
+        } catch (JsonProcessingException e) {
+            map.put(key, "Error serializing object: " + e.getMessage());
```

we should not be ignoring error. but we should pass it to the user